extensions/amp-list/0.1/test/test-amp-list.js

```diff
@@ -187,8 +187,11 @@ describes.repeated(
         function expectRender() {
```

Comment:
Are there any reasonable tests we could make for the `layout=container` case? One set of tests that comes to mind would be:
1. `attemptToFit` succeeds, styles are reset.
2. `attemptToFit` fails, height remains fixed

Reply:
Good point, added these two. Visual or e2e tests would be even better for this kind of change.